services/github/src/utils.js

```diff
@@ -0,0 +1,33 @@
+const request = require('superagent');
+const util = require('util');
+
+const setTimeoutPromise = util.promisify(setTimeout);
+
+const throttleRequest = async ({url, method, delay = 10, response = {status: 0}, attempt = 1}) => {
```

Comment:
This is a useful function -- I just needed something like it for the performance-testing framework.  Making this into a utility function in `taskcluster-client` might make a good followup and/or contributor issue.

Reply:
Yes, I was actually thinking: it would be good to have high-level `utils` thing that all the services could utilize, now that we have a monorepo